packages/eslint-config-udemy-basics/parts/stylistic-issues.js

```diff
@@ -10,7 +10,10 @@ module.exports = {
         // enforce one true brace style
```

Comment:
Remember that it's really important to write your commit messages in the right format since they automatically control the versioning and the CHANGELOG. See the bottom of https://github.com/udemy/js-tooling. You may want to rebase and change how you worded your commit message.